src/libraries/Common/src/Interop/Windows/SspiCli/NegotiationInfoClass.cs

```diff
@@ -31,7 +32,7 @@ internal static string GetAuthenticationPackageName(SafeHandle safeHandle, int n
                     string name;
                     unsafe
                     {
-                        name = Marshal.PtrToStringUni(((SecurityPackageInfo*)packageInfo)->Name);
+                        name = Marshal.PtrToStringUni(((SecurityPackageInfo*)packageInfo)->Name)!;
```

Comment:
If we declared `string name` as `string? name`, you could get rid of this suppression.